apps/graphql/src/apps/itinerary/helpers/Itineraries.js

```diff
@@ -37,10 +38,17 @@ export const mapVehicle = (type: ?string, uniqueNo: ?string) => ({
   uniqueNo: uniqueNo ?? null,
 });
 
-export const sanitizeCarrier = (routeItem: ?ApiRouteItem) => ({
-  name: null, // @TODO - value is currently missing from API endpoint response
-  code: routeItem?.airline ?? null,
-});
+export const sanitizeCarrier = (routeItem: ?ApiRouteItem) => {
+  // @TODO - value is currently missing from API endpoint response
+  // JSON file is used from https://github.com/BesrourMS/Airlines/blob/master/airlines.json
+  const airlineCode = routeItem?.airline;
+  const airline = airlines.find(airline => airline.iata === airlineCode);
+  const name = airline ? airline.name : null;
```

Comment:
const name = airline?.name

Reply:
That's the last thing and I think it's good to go :) 🚀

Reply:
thanks